OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs

```diff
@@ -79,8 +79,11 @@ public GameInfoStatsLogic(Widget widget, ModData modData, World world, OrderMana
 					checkbox.GetText = () => mo.Objectives.First().Description;
 				}
 
-				statusLabel.GetText = () => player.WinState == WinState.Won ? Accomplished :
-					player.WinState == WinState.Lost ? Failed : InProgress;
+				var failed = modData.Translation.GetString(Failed);
```

Comment:
ideally only one of the three translations is looked up. i'd be in favor for a util function `GetTranslation(modData, InProgress)` or `T(modData, key)` that can be easily used inline.

Reply:
It would have to use CashedTransforms, as GetText is called every frame. And tbf I don't think it should be a util function as it only needed in one place

Reply:
I don't think the performance gain here justifies much more complexity.